tests/check/check_h26xsigned_auth.c

```diff
@@ -2061,6 +2062,82 @@ START_TEST(no_public_key_in_sei_and_bad_public_key_on_validation_side)
 }
 END_TEST
 
+/* Test description
+ * 1. Setup a signed_video_t session
+ * 2. Set |is_start_stream| flag as true
+ * 3. Add 1 P NALU and 2 I NALU for signing that will trigger 2 SEI NALU
```

Comment:
This test is not really setup as a correct use case. BUT this gives me a good idea for how this can (should) work.
I think it should be possible for a user to (in pseudo code) do
```
sv = signed_video_create(codec);
signed_video_enable_use_of_start_of_stream_sei(sv);
signed_video_generate_start_of_stream_sei(sv);
// Add NAL Units for signing and fetch SEIs as normal. The first one will be a "golden SEI"
```

But this should be tested in a different PR, when more parts are in place. I suggest that you instead of this test should add a line in one of the helper functions that does
```
sv_rc = signed_video_is_start_of_stream(sv, nalu_data, nalu_data_size);
ck_assert_int_eq(sv_rc, SV_OK);
```